.github/workflows/deploy-apis-to-production.yml

```diff
@@ -673,7 +670,6 @@ jobs:
           valueFile: "k8s/device-monitor/values-prod.yaml"
           propertyPath: "image.tag"
           value: ${{ needs.image-tag.outputs.build_id }}
-          updateFile: true
           branch: ${{ env.DEPLOY_BRANCH }}
```

Comment:
[remove deprecated use of 'updateFile'](https://github.com/airqo-platform/AirQo-api/pull/1216/commits/8be3429e09829236c151d2881baef930f14b0220)